openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -778,17 +783,22 @@ def run(self, *, dry=False, verbose=True,
         )
 
         # 3. Create the hybrid topology
+        # a. Get softcore potential settings
+        if alchem_settings.softcore_LJ == 'gapsys':
+            softcore_LJ_v2 = True
+        elif alchem_settings.softcore_LJ == 'beutler':
+            softcore_LJ_v2 = False
```

Comment:
this will probably get fixed in the settings validation, but always safe to call `.lower()` on strings when you're doing comparisons, it's free and it'll save you a headache one day

Reply:
e.g. `if alchem_settings.softcore_LJ.lower() == 'gapsys'`

Reply:
Thanks! Changed this!